lib/reporting/command_line_options.rb

```diff
@@ -45,6 +58,17 @@ def parse!(argv, out: STDOUT)
           progress = progress_v
         end
 
+        opts.on(
+          '--slice SLICE',
+          '(optional) query slice size duration, defaults to 1w',
+        ) do |slice_v|
+          slice = CloudwatchQueryTimeSlice.parse_duration(slice_v)
+        end
+
+        opts.on('--threads THREADS', '(optional) number of threads, defaults to 5') do |threads_v|
+          threads = threads_v.to_i if threads_v.to_i.between?(1, 30)
```

Comment:
Why limit here, and why silently? If values bigger than 30 are a problem, I think we should either print a warning or throw an error

Reply:
5 just seemed reasonable? No real thought process

Reply:
okay yeah it throttles for larger than 30. what do you think the best ux here would be? if it's larger than 30, print a warning and set it to 30?

Reply:
I'd just throw an error bigger than 30 to give immediate feedback